core/src/main/java/org/elasticsearch/search/aggregations/pipeline/movavg/MovAvgPipelineAggregator.java

```diff
@@ -123,7 +131,7 @@ public InternalAggregation reduce(InternalAggregation aggregation, ReduceContext
 
                 // Some models (e.g. HoltWinters) have certain preconditions that must be met
                 if (model.hasValue(values.size())) {
-                    double movavg = model.next(values);
+                    double movavg = model.next(values, modelCoef);
```

Comment:
Can we set the `modelCoef` once on the model with a setter method before we start calculating the moving average rather than passing it into every `next()` call?